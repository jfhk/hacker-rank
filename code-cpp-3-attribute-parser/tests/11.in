6 8
<tag1 value = "v1">
<tag3 value1 = "value 1" value2 = "value 2">
</tag3>
</tag1>
<tag2>
</tag2>
tag1~value
tag1~something_not_exist
tag1.tag3~value1
tag1.tag3~value2
tag1.tag3~value3
tag1.tag2~value1
tag2~something_also_not_exist
tag3~doesnotmatter
