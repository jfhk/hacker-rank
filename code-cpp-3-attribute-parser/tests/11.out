v1
Not Found!
value 1
value 2
Not Found!
Not Found!
Not Found!
Not Found!
